tests/unit/services/authToken.test.ts

```diff
@@ -0,0 +1,96 @@
+import AxiosMockAdapter from 'axios-mock-adapter';
```

Comment:
all those service unit tests look like duplicates to the integration tests. do we really need them? whats the advantage? didnot do it in php :)

Reply:
Have you looked at our tests coverage? We will be writing _a ton_ of tests after now. 😄

Reply:
as discussed in whatsapp i see no additional value in this because from a sdk point of view you roughly test the same stuff with the only difference that you use the full production backendapi or a mocked backendapi. since all other sdks (and php) use the production backendapi we should do the same here. but if i am missing something you can always fight for it and we can rediscuss today :) otherwise please delete the "unit test" ones (with the mocked backendapi).